packages/elements-core/src/components/Docs/Article/index.tsx

```diff
@@ -2,16 +2,22 @@ import { IMarkdownViewerProps } from '@stoplight/markdown-viewer';
 import { Box } from '@stoplight/mosaic';
 import { withErrorBoundary } from '@stoplight/react-error-boundary';
 import * as React from 'react';
+import { useLocation } from 'react-router-dom';
 
+import { RouterTypeContext } from '../../../context/RouterType';
 import { MarkdownViewer } from '../../MarkdownViewer';
 import { DocsComponentProps } from '..';
 
 type ArticleProps = DocsComponentProps<IMarkdownViewerProps['markdown']>;
 
 const ArticleComponent = React.memo<ArticleProps>(({ data }) => {
+  const { pathname } = useLocation();
```

Comment:
pathname dose not return hash I believe. react-router docs says useLocations returns a hash prop.
https://github.com/remix-run/history/blob/main/docs/api-reference.md#location

Didn't test this out on our version though.

```suggestion
  const { hash } = useLocation();
```